sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -120,7 +120,7 @@ def __new__(cls, x, y):
         return super(Point, cls).__new__(cls, x, y)
 
 
-class PageRange(namedtuple("PageRange", "first_page last_page")):
+class FormPageRange(namedtuple("FormPageRange", "first_page last_page")):
     """The 1-based page range of the document.
```

Comment:
very tiny nit (please feel free to ignore): I think it would be better to have the comment as "The 1-based page range of the form" since we have form in the name and form is more specifically what we're returning the page range of.

This change would also include later parts of this docstring, i.e. changing 
:ivar int first_page: The first page number of the document
to 
:ivar int first_page: The first page number of the form

Reply:
```suggestion
    """The 1-based page range of the form.
```